### PR TITLE
Update GA course code processing

### PR DESCRIPTION
**Pull request by drewsamnick**
We are changing the set of course codes used for GA. This means that we'll need to reseed the data after this change is in prod. I've added a script that will do the reseeding. I also added code to normalize the course codes that are in the data files.

### CODE REVIEW

**Comment by drewsamnick** [comment]
[Remove "Business and Computer Science" from GA data](https://trello.com/c/cYUyjiyn/161-remove-business-and-computer-science-from-ga-data)

File: bin/oneoff/census/state-data/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Scripts for processing state data
+
+* `preprocess-ca-data.rb` - The raw file we get from CA (CoursesTaught via https://www.cde.ca.gov/ds/sd/df/filesassign.asp) has data for all courses in all schools in CA, not just CS. Rather than processing that huge file as part of seeding we just filter out the rows we don't care about. At the same time we convert it from TSV to CSV.
+* `upload-state-data.rb` - Takes a state data file and uploads it to S3 with the correctly formated key. Makes a backup of existing S3 object if one exists.
```
Comment (breville) [inline bin/oneoff/census/state-data/README.md]: nit: formatted :)

File: bin/oneoff/census/state-data/reseed-state-data.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../../dashboard/config/environment'
```
Comment (breville) [inline bin/oneoff/census/state-data/reseed-state-data.rb:3]: at this point maybe absolute path is better?

File: bin/oneoff/census/state-data/README.md (hunk 1 above)
Comment (drewsamnick) [reply bin/oneoff/census/state-data/README.md]: Oops. Fixed now.

**Comment by drewsamnick** [comment]
I realized that reseed-state-data should call `Census::StateCsOffering.seed` instead of `Census::StateCsOffering.seed_from_s3` so that it does the right thing even when using stubbed data. Fixed now.

File: bin/oneoff/census/state-data/reseed-state-data.rb (hunk 2 above)
Comment (drewsamnick) [reply bin/oneoff/census/state-data/reseed-state-data.rb:3]: Is there an absolute path that will work in all environments?

Comment (breville) [reply bin/oneoff/census/state-data/reseed-state-data.rb:3]: After a bunch of searching, apparently not.  Pegasus implemented a root `src_dir` but I don't see one for other environments.  Not to worry then.

**Pull request merged**